src/gatsby-node.js

```diff
@@ -171,7 +166,14 @@ const processData = (data, baseUrl, backlinks) => {
   // Tree hierarchy in nodes
   node.id = urlWithoutParameters(data['@id']);
   node.parent = data.parent['@id'] ? data.parent['@id'] : null;
-  node.children = data.items ? data.items.map(item => item['@id']) : [];
+
+  // Process children items, considering case of batching
+  if (node.batching) {
+    const allItems = await fetchAllItems(node.id, token);
```

Comment:
@ajayns Here you are using await in non-async function. So, you should make processData async?